From 07d9bdcb0ca54c1d58cd88e90706306a7c5102a6 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Wed, 1 Jan 2014 01:25:17 +0800
Subject: [PATCH 1/4] create directories with mkdir -p

Upstream-Status: Inappropriate [configuration]

[Patrick Ohly: also remove obsolete AM_MKINSTALLDIRS call because then
we do not need to provide it]

Signed-off-by: Mei Lei <lei.mei@intel.com>
Signed-off-by: Patrick Ohly <patrick.ohly@intel.com>
---
 configure.ac      | 3 ++-
 po/Makefile.in.in | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index c1fe224..f5ac628 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1374,7 +1374,8 @@ if test -n "$WITH_DIET_LIBC" ; then
 	INCLUDES="$INCLUDES -D_REENTRANT"
 fi
 AC_SUBST(INCLUDES)
-AM_MKINSTALLDIRS
+MKINSTALLDIRS="mkdir -p"
+AC_SUBST(MKINSTALLDIRS)
 dnl
 dnl Build CFLAGS
 dnl
diff --git a/po/Makefile.in.in b/po/Makefile.in.in
index 2ae2f2a..0badb7e 100644
--- a/po/Makefile.in.in
+++ b/po/Makefile.in.in
@@ -30,7 +30,7 @@ gettextsrcdir = $(datadir)/gettext/po
 INSTALL = @INSTALL@
 INSTALL_DATA = @INSTALL_DATA@
 MKINSTALLDIRS = @MKINSTALLDIRS@
-mkinstalldirs = $(SHELL) $(MKINSTALLDIRS)
+mkinstalldirs = $(MKINSTALLDIRS)
 
 GMSGFMT = @GMSGFMT@
 MSGFMT = @MSGFMT@
-- 
2.1.4

